src/Form/FormAnalyzer.js

```diff
@@ -93,11 +93,11 @@ class FormAnalyzer {
     evaluatePageHeadings () {
         const headings = document.querySelectorAll('h1, h2, h3, [class*="title"], [id*="title"]')
         if (headings) {
-            headings.forEach(({innerText}) => {
+            headings.forEach(({textContent}) => {
```

Comment:
What we really wanted here is `textContent`. We don't expect significant hidden text from headings and `innerText` is more expensive (triggers a reflow).